src/services/ai-assistant/ai-assistant-service.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024-present Acrolinx GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AcrolinxEndpoint } from '../../index';
+
+/**
+ * Available in Acrolinx One
+ */
+export class AIAssistantService {
+  private readonly aiAssistantBasePath = '/content-cube/assistant';
+  constructor(private readonly endpoint: AcrolinxEndpoint) {}
+
+  public getAIAssistantUrl(targetId?: string): string {
```

Comment:
- This method could be static.
- Only URL from endpoint is needed. We can just pass the URL
```
export interface AiAssistantUrlProps {
targetId?: string;
endpointUrl: string;
}
```